### PR TITLE
docs: add how to bump react-native and where to find dep-check docs

### PR DESCRIPTION
**Pull request by tido64**
### Platforms Impacted
- [x] iOS
- [x] macOS
- [x] win32 (Office)
- [x] windows
- [x] android

### Description of changes

Added docs for how to bump `react-native` and friends. And where to go to learn more about `@rnx-kit/dep-check`.

### Verification

Nothing to test.

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/android/README.md [hunk 1]
```diff
@@ -12,28 +12,53 @@ Prereqs:
...
 
 - The first time you run your project, you may get errors about missing SDKs. Android Studio usually provides quick options to resolve these issues, but you can also go to Tools->SDK Manager to manually install or update SDK platforms or tools for your project.
 
 - If you would like to debug in Android Studio, you can do the following to open the project there:
 
-```
-    # Android
-    open -a "Android Studio" .
+```sh
+# Android
```
Comment (Saadnajmi) [inline apps/android/README.md:57]: Nit: This is a typo that should say macOS. If we're already touching this, could we fix this too? Sorry about that 😅

File: apps/android/README.md [hunk 2]
```diff
@@ -12,28 +12,53 @@ Prereqs:
...
+}
+```
+
+Bump `reactNativeVersion`, and run `yarn rnx-dep-check --write`. This command
+will ensure that all relevant packages are bumped correctly.
+
+You can read more about this tool here:
+[`@rnx-kit/dep-check` design document](https://github.com/microsoft/rnx-kit/blob/main/packages/dep-check/DESIGN.md)
+
+## Troubleshooting
```
Comment (Saadnajmi) [inline apps/android/README.md:50]: What's up with the syntax highlighting here with GitHub? All of the characters have black boxes behind them for me

**Review by Saadnajmi — APPROVED** [review]

File: apps/android/README.md (hunk 2 above)
Comment (tido64) [reply apps/android/README.md:50]: Yeah, I see it too. You can try rich diff instead:

image

Comment (Saadnajmi) [reply apps/android/README.md:50]: Rich diff shows things as fine, which is why I figured it's ok.

**Pull request merged**